Loop/Managers/DeviceDataManager.swift

```diff
@@ -334,7 +350,7 @@ final class DeviceDataManager {
         self.automaticDosingStatus = automaticDosingStatus
 
         // HealthStorePreferredGlucoseUnitDidChange will be notified once the user completes the health access form. Set to .milligramsPerDeciliter until then
-        displayGlucoseUnitObservable = DisplayGlucoseUnitObservable(displayGlucoseUnit: glucoseStore.preferredUnit ?? .milligramsPerDeciliter)
+        displayGlucoseUnitObservable = DisplayGlucoseUnitObservable(displayGlucoseUnit: .milligramsPerDeciliter)
```

Comment:
why not use the user preferred unit?

Reply:
The HealthKit preferred lookup is an asynchronous call, and is generally not available at this point in app initialization. The correct value will be propagated on completion of HK authorization and glucose store initialization.